src/components/SuggestMeal.js

```diff
@@ -276,41 +249,73 @@ class SuggestMeal extends Component {
       return;
     }
 
-    var url = "/api/addMealSuggestion/";
-    console.log("gets to call fetch");
+    var url = "http://localhost:5000/api/addMealSuggestion/";
+    console.log("gets to call fetch", instructionsChip);
 
-    fetch(url, {
-      method: "POST",
-      headers: {
-        Accept: "application/json",
-        "Content-Type": "application/json",
-      },
-      body: JSON.stringify({
-        mealLabel,
-        intro,
-        servings,
-        formatted_ingredient,
-        instructionsChip,
-        ingredientStrings,
-        imgSrc,
-        readTime,
-        cookTime,
-        categoryChips,
-      }),
+    let sugestMealForm = new FormData();
+    sugestMealForm.set('mealLabel', mealLabel);
+    sugestMealForm.set('intro', intro);
+    sugestMealForm.set('servings', servings);
+    sugestMealForm.set('formatted_ingredient', JSON.stringify(formatted_ingredient));
+    sugestMealForm.set('instructionsChip', JSON.stringify(instructionsChip));
+    sugestMealForm.set('ingredientStrings', ingredientStrings);
+    sugestMealForm.set('imgSrc', imgSrc, imgSrc.name);
+    sugestMealForm.set('readTime', readTime);
+    sugestMealForm.set('cookTime', cookTime);
+    sugestMealForm.set('categoryChips', JSON.stringify(categoryChips));
+
+
+    const config = {
+      method: 'POST',
+      data: sugestMealForm,
+      url: url
+    };
+    axios(config).then(response => {
```

Comment:
why change from axios to fetch ?

Reply:
Please compare fetch  and axios. we used Formdata as body. So, we need to send this formdata into server by axios.